EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -170,14 +170,23 @@ private static int GetChargeStateMode(List<PeptideSpectralMatch> psms)
                 Dictionary<int, List<double>> hydrobophobicites = new Dictionary<int, List<double>>();
                 Dictionary<int, Tuple<double, double>> averagesCommaStandardDeviations = new Dictionary<int, Tuple<double, double>>();
```

Comment:
It would be good to change this in a later PR from the `Tuple<double,double>` structure to something like `(double name1, double name2)` so that it's a bit easier to follow the variable names. this code is pretty hard to follow